src/App.js

```diff
@@ -1,15 +1,17 @@
 import React, { Component } from 'react';
 import './App.css';
 import Banner from './components/Banner/Banner.js';
+import Map from './components/Map/Map'
```

Comment:
This import should be in routes yes?

Reply:
Yeah, good catch. I removed the import from App.js. I did not import it into routes.js because I'm not exactly sure how we'll render that on the search page. I don't "Map" is a route. Probably some thing like "Search" would render the Map component

Reply:
@tstrat